### ISSUE TITLE
The Add Account button in the vertical toolbar should be labeled /read as "Add Account Button" (NVDA)

### ISSUE THREAD

**Issue by jinglouMSFT** [microsoft/AzureStorageExplorer#1195]
**Storage Explorer Version**: 1.7.0
**Platform/OS**: All
**Architecture**: All
**Regression From**: No

**Bug description**
The Add Account button in the vertical toolbar currently is read as "Add Account". The user didn't know for sure it's clickable for actions.
